tests/integration/test_scan.py

```diff
@@ -144,6 +145,33 @@ def nested_run_dir():
     rmtree(tmp_path)
 
 
+@pytest.fixture
+def source_dirs(mock_glbl_cfg):
+    src = Path(TemporaryDirectory().name)
+    src.mkdir()
```

Comment:
Why not use `tmp_path / 'cylc-src'` instead?

Reply:
That is an option for this fixture, but not for the others above due to scoping. It made sense to maintain consistency within the file.

Reply:
I see. The `tmp_path_factory` fixture would probably be cleaner but happy to leave as is